prow/scripts/cluster-integration/helpers/encrypt-certs.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#Description: encrypts cert files from gcloud
+# The purpose of the script is to encrypt the private key and cert for HTTPS in nightly builds.
+#
+#Expected vars:
+# - KYMA_KEYRING: kyma keyring name
+# - KYMA_ENCRYPTION_KEY: encryption key name used to encrypt the files
+# - TEST_INFRA_SOURCES_DIR: Path for shout library
+
+set -o errexit
+
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/library.sh"
+
+shout "Encrypting ${DOMAIN}.build.kyma-project.io.key.encrypted"
+gcloud kms encrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}" \
+	--plaintext-file "./letsencrypt/live/${DOMAIN}/privkey.pem" \
+ 	--ciphertext-file "./letsencrypt/live/${DOMAIN}/${DOMAIN}.build.kyma-project.io.key.encrypted"
```

Comment:
Btw. instead of hardcoding paths and duplicating "logic" responsible for encrypting you can provide paths as arguments, then you can execute this script as:
```bash
encrypt-certs.sh ./letsencrypt/live/${DOMAIN}/${DOMAIN}.key.encrypted ./letsencrypt/live/${DOMAIN}/privkey.pem
```

Thanks to that script will be generic, paths will not be hardcoded, also "logic" for encrypting will not be duplicated

You can even rename this script to `encrypt.sh`. Same comment for decrypting.